CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin

```diff
@@ -1,6 +1,5 @@
 {
 	"FileVersion": 3,
-	"EngineVersion": 4.27.2,
```

Comment:
Not sure why this is removed? I saw previously @rarcif added it https://github.com/cronos-labs/play-unreal-plugin/pull/66 but there was no issue linked, so I'm not sure what the problem was

Reply:
with EngineVersion, i cannot open the project.
i'll find another solution.

Reply:
restored EngineVersion, confirmed working in mac

Reply:
changing to "4.27.2" , working
4.27.2 is not json number, cannot be read as valid json value

Reply:
log location:
~/Library/Logs/Unreal Engine/CronosPlaySdkEditor/CronosPlaySdk.log